api/pubsub.py

```diff
@@ -45,6 +59,11 @@ async def subscribe(self, channel):
             return Subscription(id=sub_id, channel=channel)
 
     async def unsubscribe(self, sub_id):
+        """Unsubscribe from a Pub/Sub channel
+
+        Unsubscribe from a channel using the provided subscription id as found
+        in a Subscription object.
```

Comment:
I would suggest,
```
        Unsubscribe from a channel asynchronously using the provided subscription id as found
        in a Subscription object and return True. Return False if subscription id can not be found 
        in `_subscriptions` dictionary. 
```

Reply:
The `_subscriptions` is a private attribute of the class, so it's an implementation detail and not something that should be shared with users.  But yes, returning True / False should be documented (or maybe the method should raise an exception).  So I'll fix that.

Reply:
@gctucker I think this is not resolved yet.